### PR TITLE
Make work/product board configurable

### PR DESCRIPTION
**Pull request by Almad**
Pull request for [Make work/product board configurable](https://trello.com/c/5qjQthn8/2482-make-work-product-board-configurable).

### CODE REVIEW

File: blackbelt/config.py [hunk 1]
```diff
@@ -23,6 +27,8 @@
     }
 # default config, overwrite/modularize
 
-config['trello']['work_board_id'] = '1KsoiV9e'
+if not config['trello']['work_board_id']:
+    config['trello']['work_board_id'] = '1KsoiV9e'
```
Comment (fosrias) [inline blackbelt/config.py]: Magic string? Could this point to a constant that is more understandable? Presume this is DEFAULT_BOARD_ID.

File: blackbelt/configure.py [hunk 2]
```diff
@@ -55,6 +83,22 @@ def configure_blackbelt():
         token_url=handle_trello.get_token_url()
     )
 
+    board_id_regexp = r"^https://trello\.com/b/(?P<id>\w+)/.*$"
+
+    get_trello_url(
+        config=config,
+        key='product_board',
+        prompt="Please provide an URL to your Product Board",
+        id_regexp=board_id_regexp
```
Comment (fosrias) [inline blackbelt/configure.py]: Curious on the injection of this? Will it ever be different?

**Comment by fosrias** [comment]
@lukas Few minor comments/questions. Mergeable as is. Presume there is another PR coming w/r to how bb t schedule list decides where it is creating cards (e.g. --board and --column)? Or do we need to use canonical names?

File: blackbelt/config.py (hunk 1 above)
Comment (Almad) [inline blackbelt/config.py]: I'd prefer it to get killed & deleted instead, in a sprint or two.

File: blackbelt/configure.py (hunk 2 above)
Comment (Almad) [inline blackbelt/configure.py]: More thinking about merging both get_functions in the future

**Comment by Almad** [comment]
@fosrias Yup, I'll continue with other cards and I don't know about columns yet, will figure that out as I go ;)

**Comment by fosrias** [comment]
@almad So at this time it is pre-mature to try out bb t schedule_list,
correct?

**Comment by Almad** [comment]
@fosrias nope, that one should work for you (for the story card -> work overview stuff)

**Pull request merged**